README.md

```diff
@@ -12,77 +12,72 @@ Fishbowl is built with [Material UI](https://material-ui.com/), [Typescript](htt
 
 # Local Development
 
-1. Clone the repo:
+## Installing
 
-```bash
-git clone git@github.com:avimoondra/fishbowl.git
-```
+1. Clone the repo:
 
-2. Install [Brew](https://brew.sh/), and:
+    ```bash
+    git clone git@github.com:avimoondra/fishbowl.git
+    ```
 
-```bash
-brew bundle
-```
+2. Install [Homebrew](https://brew.sh/), and:
 
-3. Install Node:
+    ```bash
+    brew bundle
+    ```
 
-`nvm` is used to specify the node version it requires, based on `.nvmrc` file.
+3. Install Node.js via [Node Version Manager](https://github.com/nvm-sh/nvm):
 
-```bash
-nvm install
-nvm use
-```
+    ```bash
+    nvm install
+    ```
 
 4. Install Docker
 
-Download and install [Docker](https://docs.docker.com/docker-for-mac/install/) via Docker for Mac. Further documentation can be found [here](https://docs.docker.com/engine/docker-overview/).
-
-5. Install Hasura
-
-[Hasura](https://hasura.io/) is a GraphQL server that gives you instant, realtime GraphQL APIs over Postgres, with webhook triggers on database events, and remote schemas for business logic.
+    Download and install [Docker](https://docs.docker.com/docker-for-mac/install/) via Docker for Mac. Further documentation can be found [here](https://docs.docker.com/engine/docker-overview/).
 
-```bash
-curl -L https://github.com/hasura/graphql-engine/raw/stable/cli/get.sh | bash
-```
+5. Install Hasura CLI
 
-See more detailed instructions [here](https://hasura.io/docs/1.0/graphql/manual/hasura-cli/install-hasura-cli.html)
+    [Hasura](https://hasura.io/) is a GraphQL server that gives you instant, realtime GraphQL APIs over Postgres, with webhook triggers on database events, and remote schemas for business logic.
 
-6. Install Hasura CLI
+    ```bash
+    curl -L https://github.com/hasura/graphql-engine/raw/stable/cli/get.sh | bash
+    ```
 
-See https://docs.hasura.io/1.0/graphql/manual/hasura-cli/install-hasura-cli.html
```

Comment:
This seemed to be describing the same thing as above, but let me know if I'm missing something.

Reply:
in case it changes, but 🤷 keep it short :)